docs/building-images.md

```diff
@@ -107,6 +115,11 @@ operatingSystem:
   * `disable` - Optional; List of systemd services to disable.
 * `keymap` - Optional; sets the virtual console (VC) keymap, full list via `localectl list-keymaps`. If unset, we default to
   `us`.
+* `packages` - Optional; Defines packages that need to be resolved and installed on the operating system during the combustion phase. For detailed information on how to use this configuration, see the [Installing pacakges](installing-packages.md) guide.
```

Comment:
I see why you used the phrase "need to be resolved", but I worry that won't be enough for new devs who haven't been around for the discussions to understand. Perhaps something like:

"Packages that should have their dependencies determined and pre-loaded into the built image" -- or something along those lines.